data/human/deep missions.txt

```diff
@@ -2431,3 +2431,340 @@ event "battle in arneb end"
+			`	"That is an Elf-Stone. It is why we built our Family-Cairns here. Such stones can be found on most worlds in The Deep. Indestructible, and left by the Alfar for some unknowable reason. Some, like this one, have been mostly forgotten. Others though, are important places of celebration and even worship.`
+			`	He points at a smallish cairn next to the Elf-Stone. "This, though, is what we're here for."`
+			choice
+				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`
+					goto thorcarn
+			label thorcarn
+			`	Thorleif approaches the cairn.`
+			`	Thorleif holds up the stone of his father in his palm and begins to read the runes carved upon it.`
+			`	"The day my mother gave me this stone and said she was proud of me."`
+			`	"The day my son Thorleif was born."`
+			`	"The day I struck the mother lode on Helheim."`
+			`	"The day I met <first> and he helped me find my family again."`
+			`	"The day I saw my son again, even though he would not see me."`
+			`	Thorleif sniffs, and takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
+			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now, that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, dad."`
+			`	With that Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`
+			choice
+				`	(Follow Thorleif back to the village.)`	
+			`	The two of you trudge back to the small hamlet of Grimssalr, with its people happy at having guests and long lost relatives, and to Ragnhild, happy to spend more time with Thorlief, where you feast and talk and laugh and remember.`
+			choice
+				`	(You enjoy yourself well into the night before finally succumbing to sleep.)`
+			`	In the morning the three of you catch the hovercraft back to the spaceport.`
+			`	"This place," Thorleif begins, "this world, it feels like it has forgotten its soul. Industry, and greed have taken hold, and the old happy places like Grimssalr are dying. It is good my family left here all those years ago. Norn is, and has been, a good place to be."`
+			`	Thorleif than nods to Ragnhild, "Which is also why my very new and very dear friend Ragnhild has decided to join us on the return trip to Norn." You look to Ragnhild but somehow you can't feel that you're surprised at the news.`
+			`	She smiles brightly and shrugs, "Best offer I've had in a long time." The three of you board your ship and prepare for the journey home.`
```

Comment:
```suggestion
			`	She smiles brightly. "Best offer I've had in a long time." The three of you board your ship and prepare for the journey home.`
```
You tend to use shrugging a lot, and usually it's fine, but this is probably the one spot where I think maybe leaving it out would be a good idea. I sort of get what you're going for, but by this time it could be repetitive anyways.